src/app/shared/rss-feed/rss.component.html

```diff
@@ -1,5 +1,9 @@
-<ng-container *ngIf="(isEnabled$ | async) && (hasRoute('home') || hasRoute('collections') || hasRoute('communities'))">
+<ng-container
+  *ngIf="(isEnabled$ | async) && (hasRoute('home') || hasRoute('exhibits') || hasRoute('workflow') || hasRoute('admin') || hasRoute('mydspace') || hasRoute('search') || hasRoute('collections') || hasRoute('communities'))">
```

Comment:
We reviewed this PR as a group today in the DSpace developer meeting, and had some feedback on this change.

We realize the original code already contained a few of these hasRoute() checks here, but it's a bad idea to do that: We're calling (at worst) every one of these functions, every time angular re-renders this component which could be multiple times a second. So I'd at least create a single shouldRender BehaviorSubject that checks the entire contents of this ngIf _once_ in NgOnInit, and then use that in the HTML

But since this is checking what route we're on, you could go a step further and add e.g. `enableRSS: true` as [data to those routes](https://paul-chesa.medium.com/passing-data-in-routes-in-angular-a-simple-guide-e24b814e7076). And then check for your `shouldRender` observable whether the route has that data, and whether it's enabled in the config

Reply:
I ended up going with the route data functionality as it was a lot more elegant of a fix for this so thanks for that suggestion.